build/lib/dome9ApiV2Py/dome9ApiV2Py.py

```diff
@@ -0,0 +1,315 @@
+		if outAsJson:
+			print(json.dumps(apiCall))
+		return apiCall
+
+	def getCloudAccounts(self, outAsJson=False):
+		apiCall = self.get(route='CloudAccounts')
+		if outAsJson:
+			print(json.dumps(apiCall))
+		return apiCall
+
+	def getCloudAccountID(self, ID, outAsJson=False):
+		apiCall = self.get(route='CloudAccounts/{}'.format(ID))
+		if outAsJson:
+			print(json.dumps(apiCall))
+		return apiCall
+
+	def getCloudAccountRegions(self, ID, outAsJson=False):
+		cloudAccID = self.getCloudAccountID(ID=ID)
+		apiCall = [region['region'] for region in cloudAccID['netSec']['regions']]
+		if outAsJson:
+			print(json.dumps(apiCall))
+		return apiCall
+
+	def getRoles(self, outAsJson=False):
+		apiCall = self.get(route='role')
+		if outAsJson:
+			print(json.dumps(apiCall))
+		return apiCall
+
+	def onBoardingAwsAccount(self,arn, secret, fullProtection=False, allowReadOnly=False, name=None, outAsJson=False):
+
+		data = {
+				"name":name,
+				"credentials":{
+					"arn":arn,
+					"secret":secret,
+					"type":"RoleBased"
+				},
+				"fullProtection":fullProtection,
+				"allowReadOnly": allowReadOnly
+				}
+
+		route = 'CloudAccounts'
+		apiCall = self.post(route=route, payload=json.dumps(data))
+		if outAsJson:
+			print(json.dumps(apiCall))
+
+		return apiCall
+
+	def onBoardingAzureAccount(self, subscriptionID, tenantID, clientID, clientPassword, name=None, operationMode='Read', outAsJson=False):
+
+		data = {
+			"name": name,
+			"subscriptionId": subscriptionID,
+			"tenantId": tenantID,
+			"credentials": {
+				"clientId": clientID,
+				"clientPassword": clientPassword
+			},
+			"operationMode": operationMode,
+		}
+
+		route = 'AzureCloudAccount'
+		apiCall = self.post(route=route, payload=json.dumps(data))
+		if outAsJson:
+			print(json.dumps(apiCall))
+
+		return apiCall
+
+	def updateOrganizationalUnitForAWSCloudAccount(self,cloudAccountID, organizationalUnitID=None, outAsJson=False):
+
+		data = {"organizationalUnitId": organizationalUnitID}
+
+		route = 'cloudaccounts/{}/organizationalUnit'.format(cloudAccountID)
+		apiCall = self.put(route=route, payload=json.dumps(data))
+		if outAsJson:
+			print(json.dumps(apiCall))
+
+		return apiCall
+
+	def updateOrganizationalUnitForAzureCloudAccount(self,cloudAccountID, organizationalUnitID=None, outAsJson=False):
+
+		data = {"organizationalUnitId": organizationalUnitID}
+
+		route = 'AzureCloudAccount/{}/organizationalUnit'.format(cloudAccountID)
+		apiCall = self.put(route=route, payload=json.dumps(data))
+		if outAsJson:
+			print(json.dumps(apiCall))
+
+		return apiCall
+
+	def updateRoleByID(self, roleID, roleName, permissions, outAsJson=False):
+
+		data = {
+					"name": roleName,
+					"permissions": permissions
+				}
+
+		route = 'Role/{}'.format(roleID)
+		apiCall = self.put(route=route, payload=json.dumps(data))
+		if outAsJson:
+			print(json.dumps(apiCall))
+
+		return apiCall
+
+	def getRoleByID(self, roleID, outAsJson=False):
+
+		route = 'Role/{}'.format(roleID)
```

Comment:
align all functions to use route var or not.
some uses the var for string formatting and than `self.get(route=route)` and some makes the string formatting inside the `self.get()`
personally, I would drop it